pset-iterators.js

```diff
@@ -66,6 +71,14 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+//  Get a list of presidents whos first name is James
+const presidentsNamedJames = presidents.filter(presidentObj => {
+  //  presidentObj.president.split(' ') is the president name split into an array
+  //  first name is stored at [0] index
+  return presidentObj.president.split(' ')[0] === 'James'
+})
+
+console.log(presidentsNamedJames)
```

Comment:
@sad3xoxo Great work here using the filter method, splitting each president's name to and returning the first name that starts with James.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)